### PR TITLE
Docs: Add note about colon in path

### PR DESCRIPTION
**Pull request by morigs**
As #2682 landed to master Fastify uses [find-my-way@4.0.0](https://github.com/delvedor/find-my-way/releases/tag/v4.0.0) which now has support for path with colons. This PR add small note to docs regarding paths with colons

### CODE REVIEW

File: docs/Routes.md [hunk 1]
```diff
@@ -5,18 +5,21 @@
...
+  - [Route Prefixing](#route-prefixing)
+    - [Handling of / route inside prefixed plugins](#handling-of--route-inside-prefixed-plugins)
   - [Custom Log Level](#custom-log-level)
   - [Custom Log Serializer](#custom-log-serializer)
-- [Route handler configuration](#routes-config)
-- [Route constraints](#constraints)
+  - [Config](#config)
+  - [Constraints](#constraints)
+    - [Version Constraints](#version-constraints)
+    - [Host Constraints](#host-constraints)
```
Comment (mcollina) [inline docs/Routes.md:22]: why was this changed?

Comment (morigs) [reply docs/Routes.md:22]: Good question. To be honest I don't understand. I've pulled master twice before making any changes...

Comment (mcollina) [reply docs/Routes.md:22]: I don't know, you edited those. Maybe restore this how it was before?

Comment (morigs) [reply docs/Routes.md]: I figured this out. The problem caused by extension which automatically updates table of contents based on headings. Fixed

**Review by mcollina — APPROVED** [review]
lgtm
